src/SignalR/clients/ts/signalr/src/AccessTokenHttpClient.ts

```diff
@@ -0,0 +1,61 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+import { HeaderNames } from "./HeaderNames";
+import { HttpClient, HttpRequest, HttpResponse } from "./HttpClient";
+
+/** @private */
+export class AccessTokenHttpClient extends HttpClient {
+    private _innerClient: HttpClient;
+    _accessToken: string | undefined;
+    private _accessTokenFactory: (() => string | Promise<string>) | undefined;
+
+    constructor(innerClient: HttpClient, currentAccessToken: string | undefined, accessTokenFactory: (() => string | Promise<string>) | undefined) {
```

Comment:
When is there ever a `currentAccessToken` at construction time? Tests?

Reply:
Huh, must have been left behind after a refactor... I don't see it being used anywhere